network/src/peers/peer/peer.rs

```diff
@@ -175,6 +190,78 @@ impl Peer {
     }
 
     pub(super) fn set_disconnected(&mut self) {
+        self.sync_state.reset();
         self.quality.disconnected();
     }
+
+    pub fn start_rtt_measurement(&mut self) {
+        self.quality.expecting_pong = true;
+        self.quality.last_ping_sent = Some(Instant::now());
+    }
+
+    pub fn stop_rtt_measurement(&mut self) {
```

Comment:
```suggestion
    pub fn stop_ping_measurement(&mut self) {
```
same rationale as above